Sources/SwiftKuery/Insert.swift

```diff
@@ -36,14 +36,17 @@ public struct Insert: Query {
     /// An array of `AuxiliaryTable` which will be used in a query with a WITH clause.
     public private (set) var with: [AuxiliaryTable]?
     
+    /// A flag for return ID
+    public private (set) var returnID: Bool
```

Comment:
Is there something stopping this being `public let`?

Reply:
No, will change that